mantidproject/mantid
# Fix Sphinx warnings on RHEL 7

Description of work.

The version of Pygments on RHEL seems to be case sensitive so
'Python' is not recognised as a valid lexer. 

Also fixed a few reference errors.

**To test:**

Build HTML docs and see that there are no errors.

No issue

*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
